airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -412,14 +431,78 @@ def consume_logs(
                 )
                 time.sleep(1)
 
+    def fetch_requested_container_logs(
```

Comment:
I think that we need to use the provided `post_termination_timeout` in this method as we did with `fetch_container_logs`

Reply:
Not sure I quite understand this comment. We are calling the `fetch_container_logs` from `fetch_requested_container_logs` which will take care of `post_termination_timeout` right?